content/BindingMethods/BIndingMethods.q.md

```diff
@@ -0,0 +1,1601 @@
+
+- It prints `Successfully connected to the database.` and `1JohnSmith1999-02-10M 2SamanthaDanes1999-10-12F 6AlisaElls1999-06-30F`.
+
+
+/// type=MS, answer=[2,3]
+
+Which of the following are array values in the associative array `$data`?
+
+- `students`
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[4]
+
+Which statement best describes the code on lines 4, 5, 6, and 7 of `student.php`?
+
+- It displays the elements of the associative array `$data`.
+
+- It holds two keys with one value in each key such as `1999-01-01` and `2000-01-01`.
+
+- It holds the elements of the associative array `$data` using the `array()` language construct.
+
+- It creates the associative array named `$data` with two keys `min_birth_date` and `max_birth_date`.
+
+- It accesses the elements of `$data` using the `array()` language construct and the short array syntax `[]`.
+
+
+/// type=MS, answer=[4,5]
+
+In `student.php`, which of the following are named placeholders?
+
+- `students`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `:max_birth_date`
+
+- `:min_birth_date`
+
+
+/// type=SS, answer=[1]
+
+In `student.php`, what value is associated with the named placeholder `:min_birth_date` on line 9?
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[2]
+
+In `student.php`, what value is associated with the named placeholder `:max_birth_date` on line 9?
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[5]
+
+Which statement best describes `WHERE birth_date >= :min_birth_date AND birth_date < :max_birth_date` on line 9 of `student.php`?
+
+- It returns all records of students whose `birth_date` starts with `1999-01-01` or `2000-01-01`.
+
+- It selects all columns in the `students` table related to the values `1999-01-01` and `2000-01-01`.
+
+- It selects all columns in the `students` table equivalent to the dates `1999-01-01` and `2000-01-01`.
+
+- It updates all columns in the `students` table whose `birth_date` is between the dates `1999-01-01` and `2000-01-01`.
+
+- It returns all records of students whose `birth_date` is greater than or equal to the date `1999-01-01` and less than the date `2000-01-01`.
+
+
+/// type=MS, answer=[2,3,4,5]
+
+Which statements correctly describe `$pstmt = $conn->prepare($sql)` on line 11 of `student.php`?
+
+- It executes the SQL statement.
+
+- It prepares the argument `$sql` for execution.
+
+- It returns the result set as a `PDOStatement` object.
+
+- It calls the `prepare()` method of the `$conn` object.
+
+- It assigns the `PDOStatement` object to the variable `$pstmt`.
+
+
+/// type=SS, answer=[3]
+
+In `student.php`, which of the following is the parameter identifier in the statement `$pstmt->bindParam(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR);` on line 12?
```

Comment:
Please rephrase this statement based on the changes of the `facts.md` content. Thanks.